Modules/mathmodule.c

```diff
@@ -53,6 +53,7 @@ raised for division by zero and mod by zero.
  */
 
 #include "Python.h"
+#include "pycore_bitutils.h" // _Py_bit_length()
```

Comment:
nitpick: There is a pseudo convention using using column 35 for such comments:

```suggestion
#include "pycore_bitutils.h"      // _Py_bit_length()
```